src/components/Footer/Footer.styles.js

```diff
@@ -11,6 +11,7 @@ export const StyledFooter = styled.div`
   @media screen and (min-width: 768px) {
     display: flex;
     flex-wrap: wrap;
+    padding: 1rem 2.5rem;
```

Comment:
Why did you change this?

Reply:
I made it so that the footer padding was consistent with the rest of the content on the web page as that is what the design showed :)